Dockerfile.mysql

```diff
@@ -2,13 +2,7 @@
 FROM mysql:latest
 
 # Environment variables for MySQL configuration
-ENV MYSQL_ROOT_PASSWORD=1234
-ENV MYSQL_DATABASE=app_db
-ENV MYSQL_USER=app_user
-ENV MYSQL_PASSWORD=1234
-
-# Expose port 3306 for MySQL connections
-EXPOSE 3306
-
-# Set the data directory as a volume
-VOLUME /var/lib/mysql
+ENV MYSQL_ROOT_PASSWORD="default"
```

Comment:
The `MYSQL_ROOT_PASSWORD` is set to 'default', which is a major security risk. Consider using Docker secrets or setting this variable at runtime to protect sensitive information.